src/sources/forus-platform/js/angular-1/components/Modals/ModalProductVoucherCreateComponent.js

```diff
@@ -0,0 +1,119 @@
+let ModalProductVoucherCreateComponent = function(
```

Comment:
I think you based this branch from a wrong source. This feature aswell is not part of this release yet :) maybe next